CHANGELOG.md

```diff
@@ -5,13 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog], and this project adheres to
 [Semantic Versioning].
 
-## [Unreleased]
+## [0.4.0] - 2022-11-08
```

Comment:
@jjfrench Ignore my other comment that might have arrived in your mail about 0.3.1 instead of 0.4.0. I was thinking that we added support for beams to 0.3.0, and that all you were doing was fixing the arg count bug in the shell script.

However, in addition to "Added" we need a "Fixed" section that indicates that we fixed that bug, which means you should create a "bug" issue in this repo that you can reference as being fixed. The line that you add to the "Fixed" section should not only indicate that the to-be-created bug issue has been fixed, but that 0.3.0 has been yanked because the bug makes it impossible to run a job with that version.

Reply:
For reference, see how we handled yanks in our previous repo: https://github.com/MAAP-Project/maap-documentation-examples/blob/main/gedi-subset/CHANGELOG.md

Reply:
The issue was that it was expecting 6 inputs but getting 4. It's not necessarily fixing the issue if the BEAMs version uses 7 algorithm inputs, more so going around the issue because the n_expected algorithm inputs would've been 7 either way.

What about the hotfixes/branch we were talking about last night? that seems like a valid solution to fix 0.3.0. I would just have to add another commit to develop to change the version to 0.4.0 after the doc changes

Reply:
Sorry, I'm not following, but we can go over the correct workflow next week. Fortunately, it's not a big deal here, but I do want to make sure we document the workflow to avoid confusion all around.